### ISSUE TITLE
Not convinced PausableBuffered works correctly

### ISSUE THREAD

**Issue by rpeach-sag** [SoftwareAG/apama-rxepl#30]
```javascript
any discard := Observable.interval(1.0)
	.pausableBuffered(Observable.interval(2.0))
	.subscribe(Subscriber.create().onNext(printValue));
```

seems to emit in batches rather than buffering/not-buffering

**Comment by rpeach-sag** [comment]
Fix with #29

**Comment by rpeach-sag** [comment]
Seems fine:

```javascript
ISubscription s := Observable.interval(1.0)
	.pausableBuffered(Observable.interval(5.0)
	.timestamp()
	.subscribe(Subscriber.create().onNext(printValue));
```

```
2018-09-23 17:24:41.702 CRIT  [988] - com.industry.rx_epl.RxEpl [1] com.industry.rx_epl.TimestampedValue(1537719881.7,any(integer,0))
2018-09-23 17:24:41.702 CRIT  [988] - com.industry.rx_epl.RxEpl [1] com.industry.rx_epl.TimestampedValue(1537719881.7,any(integer,1))
2018-09-23 17:24:41.703 CRIT  [988] - com.industry.rx_epl.RxEpl [1] com.industry.rx_epl.TimestampedValue(1537719881.7,any(integer,2))
2018-09-23 17:24:41.703 CRIT  [988] - com.industry.rx_epl.RxEpl [1] com.industry.rx_epl.TimestampedValue(1537719881.7,any(integer,3))
2018-09-23 17:24:41.703 CRIT  [988] - com.industry.rx_epl.RxEpl [1] com.industry.rx_epl.TimestampedValue(1537719881.7,any(integer,4))
2018-09-23 17:24:41.703 CRIT  [988] - com.industry.rx_epl.RxEpl [1] com.industry.rx_epl.TimestampedValue(1537719881.7,any(integer,5))
2018-09-23 17:24:42.704 CRIT  [988] - com.industry.rx_epl.RxEpl [1] com.industry.rx_epl.TimestampedValue(1537719882.7,any(integer,6))
2018-09-23 17:24:43.702 CRIT  [988] - com.industry.rx_epl.RxEpl [1] com.industry.rx_epl.TimestampedValue(1537719883.7,any(integer,7))
2018-09-23 17:24:44.705 CRIT  [988] - com.industry.rx_epl.RxEpl [1] com.industry.rx_epl.TimestampedValue(1537719884.7,any(integer,8))
2018-09-23 17:24:51.703 CRIT  [988] - com.industry.rx_epl.RxEpl [1] com.industry.rx_epl.TimestampedValue(1537719891.7,any(integer,9))
2018-09-23 17:24:51.703 CRIT  [988] - com.industry.rx_epl.RxEpl [1] com.industry.rx_epl.TimestampedValue(1537719891.7,any(integer,10))
2018-09-23 17:24:51.703 CRIT  [988] - com.industry.rx_epl.RxEpl [1] com.industry.rx_epl.TimestampedValue(1537719891.7,any(integer,11))
2018-09-23 17:24:51.703 CRIT  [988] - com.industry.rx_epl.RxEpl [1] com.industry.rx_epl.TimestampedValue(1537719891.7,any(integer,12))
2018-09-23 17:24:51.703 CRIT  [988] - com.industry.rx_epl.RxEpl [1] com.industry.rx_epl.TimestampedValue(1537719891.7,any(integer,13))
2018-09-23 17:24:51.703 CRIT  [988] - com.industry.rx_epl.RxEpl [1] com.industry.rx_epl.TimestampedValue(1537719891.7,any(integer,14))
2018-09-23 17:24:51.703 CRIT  [988] - com.industry.rx_epl.RxEpl [1] com.industry.rx_epl.TimestampedValue(1537719891.7,any(integer,15))
2018-09-23 17:24:52.703 CRIT  [988] - com.industry.rx_epl.RxEpl [1] com.industry.rx_epl.TimestampedValue(1537719892.7,any(integer,16))
2018-09-23 17:24:53.707 CRIT  [988] - com.industry.rx_epl.RxEpl [1] com.industry.rx_epl.TimestampedValue(1537719893.7,any(integer,17))
```
